dev/fattest.simplicity/src/componenttest/custom/junit/runner/FATRunner.java

```diff
@@ -622,46 +622,53 @@ private String getFFDCHeader(RemoteFile ffdcLogFile) throws Exception {
     public List<String> getExpectedFFDCAnnotationFromTest(FrameworkMethod m) {
 
         ArrayList<String> annotationListPerClass = new ArrayList<String>();
-        ExpectedFFDC ffdc = m.getAnnotation(ExpectedFFDC.class);
-        if (ffdc != null) {
-            String[] exceptionClasses = ffdc.value();
-            for (String exceptionClass : exceptionClasses) {
-                annotationListPerClass.add(exceptionClass);
+
+        Set<ExpectedFFDC> ffdcs = new HashSet<ExpectedFFDC>();
+        ffdcs.add(m.getAnnotation(ExpectedFFDC.class));
+        for (ExpectedFFDC ffdc : ffdcs) {
+            if (ffdc != null) {
+                if (ffdc.repeatAction().equals(ExpectedFFDC.ALL_REPEAT_ACTIONS) || RepeatTestFilter.CURRENT_REPEAT_ACTION.equals(ffdc.repeatAction())) {
```

Comment:
`CURRENT_REPEAT_ACTION` will be null when no `RepeatTests` are defined, so we need to account for null here